internal/commands/status.go

```diff
@@ -38,16 +38,21 @@ func runStatus(dockerCli command.Cli, claimName string, opts credentialOptions)
 	if err != nil {
 		return err
 	}
-	targetContext := getTargetContext(opts.targetContext, dockerCli.CurrentContext())
-	bind, err := requiredClaimBindMount(c, targetContext, dockerCli)
+	opts.SetDefaultTargetContext(dockerCli)
+	bind, err := requiredClaimBindMount(c, opts.targetContext, dockerCli)
 	if err != nil {
 		return err
 	}
 	driverImpl, errBuf, err := prepareDriver(dockerCli, bind, nil)
 	if err != nil {
 		return err
 	}
-	creds, err := prepareCredentialSet(targetContext, dockerCli.ContextStore(), c.Bundle, opts.credentialsets)
+	if err := mergeBundleParameters(&c,
+		withSendRegistryAuth(opts.sendRegistryAuth),
```

Comment:
Why do we need to send the registry auths for the `status` command 🤔 ?
I thought these auths were needed to pull private images, but for the status, we already have an installation and don't need to pull the images right? Or maybe I'm missing something?

Reply:
Our current implementation of `status` doesn't happen to need them. One could image e.g. that `status` might want to be able to report on whether a given deployment was running the latest images, which would need creds to do so.

However even before that happens I think that in the abstract sense most of the actions should offer a consistent set of functionality for things like this for consistency. It's also not impossible that 3rd parties might choose to produce invocation images which integrate with docker app and want to use it.

Reply:
Well, it sounds like a little YAGNI 😅Any thoughts @simonferquel @chris-crone ?

Reply:
If there is a deterministic way to tell, from the specification (not merely from the current implementation), whether a given action must or must not have these creds provided then I'm happy to implement that, otherwise I think it is better to err on the side of alllowing the option for the user.

Note that not passing these creds here is actually more work than passing them, due to the way I've handled `prepareCredentialSet` wrt the `credentialOptions` struct. I was actually considering applying the same pattern of automation to `mergeBundleParameters` too.

Reply:
Well, if it really needs more work to remove it, then keep it, and as the command line flags are still optional, if the user doesn't fill them then it won't error out, right?
It's just weird to have no-op flags on a command.

Reply:
Nothing here will be a no-op.

If the user doesn't give any option → no registry creds are sent
If the user does give the `--with-registry-creds` → registry creds are sent.

Reply:
no-op in a sense that even if we send the registry creds they won't be used by the `status` command. So either setting the flag or not, the result will be the same. That's what I call "no-op".

Reply:
I see, yes, it's a no-op with our current invocation base image, we can't say what it will do with some other invocation image.

Reply:
Oh yeah, good point, sorry I forgot it's also a generic cnab client. Then 👍